doc/news/changes/minor/20210228Munch

```diff
@@ -0,0 +1,5 @@
+New: A new class GridGenerator::MarchingCubeAlgorithm has been added. It
```

Comment:
Since it doesn't create a new Triangulation, it shouldn't be in `GridGenerator`. Maybe `GridTools` would be better?